### PR TITLE
Character limit set, history design issues and other design issues fixed

### PR DESCRIPTION
**Pull request by niravpd8000**

### CODE REVIEW

File: src/Components/Dashboard/CreateNFT/createNft.js [hunk 1]
```diff
@@ -408,6 +428,7 @@ const CreateNft = (props) => {
                   name="title"
                   value={details.title}
                   onChange={inputEvent}
+                  onPaste={(event => inputEvent({ target: {name: "title", value: event.clipboardData.getData('Text').substring(0, 60)}}))}
```
Comment (biswanathsahoo) [inline src/Components/Dashboard/CreateNFT/createNft.js:431]: Please define this 60 in a constant

File: src/Components/Dashboard/CreateNFT/createNft.js [hunk 2]
```diff
@@ -418,6 +439,7 @@ const CreateNft = (props) => {
                   rows={5}
                   name="description"
                   value={details.description}
+                  onPaste={(event => inputEvent({ target: {name: "description", value: event.clipboardData.getData('Text').substring(0, 100)}}))}
```
Comment (biswanathsahoo) [inline src/Components/Dashboard/CreateNFT/createNft.js:442]: Also, put this 100 value in a constant

File: src/layout/DetailRoute.js [hunk 3]
```diff
@@ -41,23 +43,24 @@ const Layout = ({ children }) => {
...
                           </div>
                       ) :
                       (
                           <img
-                              src={nft__detail?.image}
-                              alt={nft__detail.name}
+                              src={nft__detail?.file_url}
+                              alt={nft__detail?.title}
                           />
                       )}
```
Comment (biswanathsahoo) [inline src/layout/DetailRoute.js:65]: Please move these conditional code to a separate component as it being used in other component as well

**Review by biswanathsahoo — CHANGES_REQUESTED** [review]

**Comment by sparkyrider** [comment]
@niravpd8000 any updates on this? This is pending merge

**Pull request closed without merging**